arm/Microsoft.DocumentDB/databaseAccounts/.parameters/mongodb.parameters.json

```
@@ -3,7 +3,7 @@
     "contentVersion": "1.0.0.0",
     "parameters": {
         "name": {
-            "value": "sxx-az-cdb-x-002"
+            "value": "sxx-az-cdb-mngdb-001"
```

```suggestion
            "value": "sxx-az-cdb-mondodb-001"
```

or rename the parameter file to mngdb.parameters.json